code/datums/abilities/lizard.dm

```diff
@@ -0,0 +1,155 @@
+/mob/living/carbon/human/proc/give_lizard_powers()
+	if (ishuman(src)) // not4long
+		var/datum/abilityHolder/lizard/A = src.get_ability_holder(/datum/abilityHolder/lizard)
+		if (A && istype(A))
+			return
+		var/datum/abilityHolder/lizard/W = src.add_ability_holder(/datum/abilityHolder/lizard)
+		W.addAbility(/datum/targetable/lizardAbility/colorshift)
+		W.addAbility(/datum/targetable/lizardAbility/colorchange)
+	else return
+
+/mob/living/carbon/human/proc/remove_lizard_powers()
+	if (ishuman(src))
+		var/datum/abilityHolder/lizard/W = src.get_ability_holder(/datum/abilityHolder/lizard)
+		if (W && istype(W))
+			W.removeAbility(/datum/targetable/lizardAbility/colorshift)
+			W.removeAbility(/datum/targetable/lizardAbility/colorchange)
+			src.remove_ability_holder(/datum/abilityHolder/lizard)
+	else return
+
+/datum/abilityHolder/lizard
+	topBarRendered = 1
+	points = 0
+	regenRate = 0	// eat some kidneys or something
+	pointName = "Metachroitic Factor"
+
+/datum/targetable/lizardAbility
+	icon = 'icons/mob/genetics_powers.dmi'
+	icon_state = "lizard"
+	cooldown = 0
+	last_cast = 0
+	targeted = 0
+	preferred_holder_type = /datum/abilityHolder/lizard
+	var/mob/living/carbon/human/L
+
+	onAttach(datum/abilityHolder/H)
+		. = ..()
+		if(ishuman(holder.owner))
+			L = holder.owner
+		return
+
+	proc/fix_colors(var/hex)
```

Comment:
describe the function of this proc... i might know what it does but others won't

Reply:
Thought I already described them, but I guess that was another pr. Added a comment.